packages/cli/internal/pkg/cli/configure_format.go

```diff
@@ -30,8 +30,8 @@ func newFormatContextOpts(vars formatContextVars) (*formatContextOpts, error) {
 	}, nil
 }
 
-func (o *formatContextOpts) Validate() error {
-	if o.formatContextVars.format == "" {
+func (o *formatContextOpts) Validate(args []string) error {
+	if len(args) == 0 {
```

Comment:
This should be != 1 with a message similar to "a single format value must be provided"